src/containers/MoviesListContainer/MoviesListContainer.js

```diff
@@ -1,93 +1,71 @@
-import React, { Component } from "react";
+import React, {useState} from "react";
 import MoviesList from "../../components/MoviesList/MoviesList.js";
 import Modal from "../../components/Modal/Modal.js";
 import DeleteMovieModal from "../../components/DeleteMovieModal/DeleteMovieModal.js";
 import EdiMovieModal from "../../components/EditMovieModal/EditMovieModal.js";
 import movies from "./movies.js";
 
-class MoviesListContainer extends Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      isDeleteModalShown: false,
-      isEditModalShown: false,
-      movieToDeleteId: null,
-      movieToEditId: null,
-    };
-    this.showDeleteModal = this.showDeleteModal.bind(this);
-    this.hideDeleteModal = this.hideDeleteModal.bind(this);
-    this.deleteMovie = this.deleteMovie.bind(this);
-    this.showEditModal = this.showEditModal.bind(this);
-    this.hideEditModal = this.hideEditModal.bind(this);
-    this.editMovie = this.editMovie.bind(this);
-  }
+export default function MoviesListContainer() {
+  const [isDeleteModalShown, setIsDeleteModalShown] = useState(false);
+  const [isEditModalShown, setIsEditModalShown] = useState(false);
+  const [movieToDeleteId, setMovieToDeleteId] = useState(null);
+  const [movieToEditId, setMovieToEditId] = useState(null);
```

Comment:
It's just a proposal, but you could replace this logic on 'useReducer' hook just to get more practice with that.